index.js

```diff
@@ -5,6 +5,306 @@ const portfolio = document.querySelector('#portfolio-mobile');
+    Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+    when an unknown printer took a galley of type and scrambled it 1960s with the relea
+    Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+    Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+    when an unknown printer took a galley of type and scrambled it 1960s with the relea`,
+  },
+];
+
+const cardWorkMobile = (items) => `
+  <article class="card-work-mobile">
+    <img src="assets/img/closedetailbtn.svg" alt="close detail mobile" class="close-detail-mobile">
+    <img src=${arrayCards[items].url} alt="details image" class="card-image-details-mobile">
+    <h2 class="card-name-mobile">${arrayCards[items].name}</h2>
+    <ul class="card-tags">
+      <li>${arrayCards[items].tecnologies.tech1}</li>
+      <li>${arrayCards[items].tecnologies.tech2}</li>
+      <li>${arrayCards[items].tecnologies.tech3}</li>
+    </ul>
+    <p class="card-description-mobile">${arrayCards[items].description}</p>
+    <div class="card-button-mobile">
+      <a href=${arrayCards[items].linkVersion} class="livebtn-mobile">
+        See Live
+        <img src="assets/img/liveicon.svg" alt="live icon">
+      </a>
+      <a href=${arrayCards[items].linkSource} class="sourcebtn-mobile">
+        See Source
+        <img src="assets/img/sourceicon.svg" alt="source icon">
+      </a>
+    </div>
+    <script src="index.js"></script>
+  </article>
+`;
+
+const cardWorkDesktop = (items1) => `
+  <article class="card-work-desktop">
+    <img src="assets/img/Icon.png" alt="close detail desktop" class="close-detail-desktop">
+    <img src=${arrayDesktop[items1].url} alt="details image" class="card-image-details-desktop">
+    <div class="popup-title-desktop">
+    <h2 class="card-name-desktop">${arrayDesktop[items1].name}</h2>
+    <div class="card-button-desktop">
+       <div class="btn-container">
+        <a href=${arrayDesktop[items1].linkVersion} class="livebtn-desktop">
+          See Live
+          <img src="assets/img/liveicon.svg" alt="live icon">
+        </a>
+        </div>
+        <div class="btn-container">
+          <a href=${arrayDesktop[items1].linkSource} class="sourcebtn-desktop">
+            See Source
+            <img src="assets/img/sourceicon.svg" alt="source icon">
+          </a>
+        </div>
+    </div>
+    </div>
+    <ul class="card-tags">
+      <li>${arrayDesktop[items1].tecnologies.tech1}</li>
+      <li>${arrayDesktop[items1].tecnologies.tech2}</li>
+      <li>${arrayDesktop[items1].tecnologies.tech3}</li>
+      <li>${arrayDesktop[items1].tecnologies.tech4}</li>
+      <li>${arrayDesktop[items1].tecnologies.tech5}</li>
+      <li>${arrayDesktop[items1].tecnologies.tech6}</li>
+    </ul>
+    <p class="card-description-desktop">${arrayDesktop[items1].description}</p>
+    <script src="index.js"></script>
+  </article>
+`;
```

Comment:
- [ ] The popup on the desktop version also has some sizing problems like the mobile version. Please reduce the size of some elements like the image and description so the whole popup can fit in the screen. It is not a good user experience to have to scroll up and down to see the content of a popup window.
 - [ ] Also the gray overlay is missing. Please implement the overlay to hide the elements below the popup window like the Figma template.
 - Your popup:
 - 
<img width="1246" alt="image" src="https://user-images.githubusercontent.com/61919617/216109361-73aa89ed-90c1-4440-81ec-7471dffcf6b9.png">

 - Figma template:
 - 
<img width="556" alt="image" src="https://user-images.githubusercontent.com/61919617/216109473-96a3094e-a4c0-486e-b798-6def3f8ac01d.png">